README.md

```diff
@@ -184,20 +234,63 @@ To run tests and code coverage, use the following command:
 By default, a [Husky](https://github.com/typicode/husky) pre-commit hook is defined in `package.json`, which runs linting, testing, and code coverage before the commit is made. If either the linting, testing, or code coverage fails, the pre-commit hook prevents the changes from being committed. Review the results and fix the problems, or modify the thresholds to allow the commit to succeed.
 
 ### Experience tests
+
 You can verify the state of your locally running application by using the Selenium UI test script that is included in the `scripts` directory.
 
-### Debugging your local app
+## Enhancing your app with additional services
 
-You can build and debug your app locally by using the following command:
+### Adding a service to your app
+To add additional services to your app, do the following:
 
 ```bash
-ibmcloud dev build --debug
-ibmcloud dev debug
+ibmcloud dev edit
 ```
 
-## Deploying changes to IBM Cloud
+Select the "Manage services" option, then follow the prompts to select an existing service or provision a new one.  The terminal output will tell you which files were created or need to be merged.
 
-Simply push to the master branch of your remote repo to initiate a new build/deploy cycle. After a few minutes, your modified app is running in the cloud. You can check the progress of the deployment by visiting the pipeline details view.
+### Adding service dependencies and code into your app
+
+Your service will likely have an SDK library to make programming easier.  Add the service library as a dependency in your package.json file, and add lines of code to initialize and use your new service.  See your service documentation for details.
+
+After that is done, build and run your app locally to test the connection to the service.   Note that by using `ibm cloud edit` to add the service, you get a local credential file and you don't need to add credentials manually to your local environment.
```

Comment:
I think this should be `ibmcloud dev edit`

Reply:
fixed